images/scripts/services.setup

```diff
@@ -122,44 +122,58 @@ chmod 755 /root/run-samba-domain
 #
 #############################
 cat <<EOF > ./Dockerfile
-FROM quay.io/fedora/fedora:35
-RUN dnf -y install systemd && dnf clean all
+FROM quay.io/fedora/fedora:36
+RUN dnf -y install systemd && dnf clean all && \
+  (cd /lib/systemd/system/sysinit.target.wants/ ; for i in * ; do [ \$i == systemd-tmpfiles-setup.service ] || rm -f \$i ; done) ; \
+  rm -f /lib/systemd/system/multi-user.target.wants/* ;\
+  rm -f /etc/systemd/system/*.wants/* ;\
+  rm -f /lib/systemd/system/local-fs.target.wants/* ; \
+  rm -f /lib/systemd/system/sockets.target.wants/*udev* ; \
+  rm -f /lib/systemd/system/sockets.target.wants/*initctl* ; \
+  rm -f /lib/systemd/system/basic.target.wants/* ;\
+  rm -f /lib/systemd/system/anaconda.target.wants/*
+VOLUME ["/sys/fs/cgroup"]
+CMD ["/sbin/init"]
 EOF
 
 podman build --tag fedora_systemd .
 
-podman run --name=candlepin --uts=host --publish=8443:8443 --privileged --detach -t fedora_systemd /usr/sbin/init
+podman run --name=candlepin --uts=host --publish=8443:8443 --privileged --detach -t fedora_systemd
 
 podman exec -i candlepin sh -eux <<EOF
 # wait until booted
 systemctl start multi-user.target
 # ensure latest ca-certificates
 dnf update -y
 # Install dependencies
-dnf -y install ansible libselinux-python3 python3-firewall git-core openssl sudo which hostname initscripts gettext selinux-policy
+dnf -y install ansible libselinux-python3 python3-firewall git-core openssl sudo which hostname initscripts gettext selinux-policy acl
 
 useradd candlepin
 echo 'candlepin ALL=(ALL) NOPASSWD: ALL' > /etc/sudoers.d/candlepin
 EOF
 
 podman exec -i -u candlepin candlepin sh -eux <<EOF
-ansible-galaxy install rvm.ruby
+ansible-galaxy collection install ansible.posix community.general community.mysql community.postgresql
```

Comment:
Hmm, more moving targets.. OOI, what happens when this isn't done? Does the candlepin role fail, or specify its own dependency? Can these community roles be versioned somehow, ore are they guaranteed to be API stable? (something like "community.general" doesn't induce too much faith in me..)

Also, *two* databases? It feels like this will eat up even more RAM..

Reply:
The newer version of the "candlepin" role uses the FQMN (Fully Qualified Module Names) for the modules in external collections; because of that, we need to install those collections so they are available (see `ansible-galaxy collection list`, for example). IIRC ansible in Fedora 35 does not have a "that good" support for collections (which is also why the current pinning exists).

The "community.general" collection is the largest/most available collection, and it contains modules that used to be part of "monolithic ansible" in the past. They maintain compatibility, so it is OK to use this collection (and the other mentioned) unconditionally.

> Also, _two_ databases? It feels like this will eat up even more RAM..

This is merely installing a collection, so resolving the role in the "candlepin" role works. In practice:
- we have been deploying two databases already, because the "candlepin" role has as defaults `cp_configure_postgresql: true` and `cp_configure_mariadb: true`
- this change sets `cp_configure_mariadb: false`, so MariaDB is no more deployed (mentioned also in the commit message)